Sources/Testing/EntryPoints/EntryPoint.swift

```diff
@@ -192,9 +192,42 @@ func parseCommandLineArguments(from args: [String]) throws -> __CommandLineArgum
   // Do not consider the executable path AKA argv[0].
   let args = args.dropFirst()
 
+#if !SWT_NO_FILE_IO
+#if canImport(Foundation)
+  // Configuration for the test run passed in as a JSON file (experimental)
+  //
+  // This argument should always be the first one we parse.
+  //
+  // NOTE: While the output event stream is opened later, it is necessary to
+  // open the configuration file early (here) in order to correctly construct
+  // the resulting __CommandLineArguments_v0 instance.
+  if let configurationIndex = args.firstIndex(of: "--experimental-configuration-path"), configurationIndex < args.endIndex {
```

Comment:
These checks should be `index + 1 < endIndex` to verify the next argument exists.

Reply:
:kermit-face:

fixed